framework/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Yii Framework 2 Change Log
 - Enh #11139: `yii\validators\EachValidator` injects specific attribute value in error message parameters (silverfire)
 - Enh #11187: migrate command now generates phpdoc for table migrations (Faryshta)
 - Enh: Added `StringHelper::countWords()` that given a string returns number of words in it (samdark)
+- Enh: migrate command can create foreign keys.
```

Comment:
@samdark thats why it contains table1 and table2. But I mean, the prefix notation is properly used there? or should I use `{{%fk-table1-table2}}` ?